strawberryfields/ops.py

```diff
@@ -1052,6 +1053,23 @@ def _apply(self, reg, backend, shots=1, **kwargs):
         return backend.measure_fock(reg, shots=shots, select=self.select, **kwargs)
 
 
+class MeasureThreshold(Measurement):
+    """:ref:`photo_detection`: measures a set of modes in the thresholded Fock basis, i.e.
```

Comment:
just curious, where does `photo_detection` resolve to in docs?

Reply:
Now that I think about it doesn't. Photodetection is however briefly mention in the sf paper in Quantum.